control/control.go

```diff
@@ -0,0 +1,104 @@
+package control
+
+import (
+	"encoding/json"
+	"errors"
+	"net"
+	"net/http"
+	"os"
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	SOCKET_TYPE = "unix"
```

Comment:
Although every sane language wants the constants in all-caps, in golang capitalization effects whether we export the variable, so we'll want this to be all lower-case as the linter wants.